### PR TITLE
Adding ppc64le support parameters

### PR DESCRIPTION
**Pull request by Sunidhi-Gaonkar1**
##### SUMMARY

Added parameters to build docker image for ppc64le platform.

##### ISSUE TYPE

 - New or Enhanced Feature

##### AWX VERSION

```
awx: 21.5.1.dev0+gff49cc5636.d20220825
```

### CODE REVIEW

File: tools/ansible/roles/dockerfile/templates/Dockerfile.j2 [hunk 1]
```diff
@@ -46,6 +46,12 @@ RUN dnf -y update && dnf install -y 'dnf-command(config-manager)' && \
 
 RUN pip3 install virtualenv build
 
+RUN dnf -y update && dnf install -y wget && \
+    wget https://static.rust-lang.org/dist/rust-1.41.0-powerpc64le-unknown-linux-gnu.tar.gz && \
```
Comment (shanemcd) [inline tools/ansible/roles/dockerfile/templates/Dockerfile.j2:51]: - Why is this needed?
- Why are you hardcoding ppc64le?

Comment (Sunidhi-Gaonkar1) [reply tools/ansible/roles/dockerfile/templates/Dockerfile.j2:51]: The rust compiler is required to install cryptography package on ppc64le.

Comment (shanemcd) [reply tools/ansible/roles/dockerfile/templates/Dockerfile.j2:51]: I just noticed the `if [ $(uname -m) = ppc64le ]` part. Can we rework this a bit?

Unless I'm missing something, `tini_architecture` and `kubectl_architecture` look exactly the same. Maybe we can combine them into a single `image_architecture` variable? You can then use that variable in a jinja conditional in Dockerfile.j2. See that file for other examples of conditionals.

Comment (Sunidhi-Gaonkar1) [reply tools/ansible/roles/dockerfile/templates/Dockerfile.j2:51]: I have made the changes, Please review.

Comment (Sunidhi-Gaonkar1) [reply tools/ansible/roles/dockerfile/templates/Dockerfile.j2:51]: @shanemcd Please review.

Comment (shanemcd) [reply tools/ansible/roles/dockerfile/templates/Dockerfile.j2:51]: Nice. Thank you.

**Comment by shanemcd** [comment]
Sorry for the delayed response. Thanks for making those changes. I've fixed one of the failing CI checks. There's one more than requires a code change:

```
./tools/ansible/roles/dockerfile/defaults/main.yml
[70](https://github.com/ansible/awx/actions/runs/3036848463/jobs/5020943356#step:8:71)
  12:136    error    no new line character at the end of file  (new-line-at-end-of-file)
```

Once this is addressed we can merge this.

**Comment by Sunidhi-Gaonkar1** [comment]
Thank you for fixing the failing CI check! I have added a new line at the end of file.

**Comment by Sunidhi-Gaonkar1** [comment]
@shanemcd Please review.

**Review by shanemcd — APPROVED** [review]
